tests/E2E/test/campaigns/full/02_product/14_options_in_catalog_page.js

```diff
@@ -32,6 +32,14 @@ scenario('Check the options in the catalog page', () => {
         .then(() => client.waitForVisibleAndClick(ProductList.action_preview_button.replace('%POS', 1)))
         .then(() => client.switchWindow(1));
     });
+    test('should click on the "Preview" link', () => {
+      return promise
+        .then(() => {
+          if (global.ps_mode_dev) {
+            client.waitForExistAndClick(AddProductPage.preview_link)
+          }
+        })
+    });
```

Comment:
I see this code written two times. You should move this code in a common secnario file, and call it from here.